qcschema/dev/dev_schema.py

```diff
@@ -10,17 +10,16 @@
 
 # The base schema definition
 base_schema = {
-    "$schema": "http://json-schema.org/draft-06/schema#",
-    "name": "QC_JSON",
-    "version": "0.1.dev",
-    "url": "http://schema_host.org/schemas/0/something.schema",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "name": "qc_schema_input",
+    "version": "dev",
```

Comment:
wouldn't hurt to make this `1.dev`, so at least one knows what version you're approaching. scrupulously, this should be `1.devN` where `N` is git commits past `0` tag, but `1.dev` will do. :-)

Reply:
Sure, fixed. I didn't add versioner or the like to keep it with the simple versioning philosophy.